shared/src/common/enums/validation/worker/worker-create-user-validation-rule.enum.ts

```diff
@@ -0,0 +1,10 @@
+const WorkerCreateUserValidationRule = {
```

Comment:
```suggestion
const WorkerValidationRule = {
```
don't forget about file name